.circleci/config.yml

```diff
@@ -40,6 +40,13 @@ jobs:
             # CircleCI and Nix sandboxing don't play nice. See
             # https://discourse.nixos.org/t/nixos-on-ovh-kimsufi-cloning-builder-process-operation-not-permitted/1494/5
             mkdir -p /etc/nix && echo "sandbox = false" > /etc/nix/nix.conf
+            nix-env -iA cachix -f https://github.com/NixOS/nixpkgs/tarball/db557aab7b690f5e0e3348459f2e4dc8fd0d9298
+      - run:
+          name: Run cachix
+          command: |
+            cachix use tweag
+            cachix push tweag --watch-store
```

Comment:
Does this also set up `cachix` as a binary substitutor that nix uses?

Reply:
`cachix use` does, it writes to `nix.conf`.

Reply:
Ah, that’s … interesting.